apps/lsdeer/lsdeer.yml

```diff
@@ -0,0 +1,12 @@
+name: lsdeer
+description: Customizable file manager.
+website: 'https://lsdeer.vercel.app/'
+repository: 'https://github.com/AlexanderPershin/lsdeer'
+screenshots:
+  - {
```

Comment:
you don't need brackets here

Reply:
Thank you for the review. I've just made all changes you requested. Please let me know if anything is still wrong

Reply:
It's all good to me! Thank you for your quick reply 👍